satpy/readers/mersi_l1b.py

```diff
@@ -179,12 +181,24 @@ def _get_bt_dataset(self, data, calibration_index, wave_number):
         else:
             # new versions of pyspectral can do dask arrays
             data.data = bt_data
+
+        # Some BT bands seem to have 0 in the first 10 columns
+        # and it is an invalid measurement, so let's mask
+        data = data.where(data != 0)
```

Comment:
Any idea if this is for all MERSI files (mersi 2 and LL)? Is it something that might be fixed? Could we put an `if` statement around this to only apply it for LL if that is the case?

Reply:
This was in the original MERSI-2 reader, and I have seen it occur with MERSI-2 files.

Haven't seen it happen with MERSI-LL, but then I've only examined a relatively small number of files so I can't definitively say that it won't happen with this instrument too!

Reply:
Ok, let's leave it here then.

Reply:
Ok one issue I see with this is that the previous version of this was applied *after* the correction coefficients were applied. I guess since the data is already supposed to be BTs (and not radiances) then this is probably still OK, but I was wondering if it might be "more" accurate to have it after the coefficients are applied?

Reply:
Changed it back for consistency.